i18n/ko.toml

```diff
@@ -0,0 +1,204 @@
+
+# homepage copy
+
+## homepage intro
+[main_title]
+other = "**The** **package manager** **for Kubernetes**"
+
+[main_subtitle]
+other = "Helm is the best way to find, share, and use software built for [Kubernetes](https://kubernetes.io)."
+
+## homepage about helm
+[main_what_is_helm_title]
+other = "What is Helm?"
+
+[main_what_is_helm_description]
+other = """
+Helm helps you manage Kubernetes applications — Helm Charts help you define, install, and upgrade even the most complex Kubernetes application. 
+
+Charts are easy to create, version, share, and publish — so start using Helm and stop the copy-and-paste.
+
+Helm is an incubator project in the [CNCF](https://cncf.io) and is maintained by the [Helm community](https://github.com/helm/community).
+
+### Learn more:
+
+* [Helm Architecture](./docs/topics/architecture/)
+* [Quick Start Guide](./docs/intro/quickstart/)
+* [Video: An Introduction to Helm](https://www.youtube.com/watch?v=Zzwq9FmZdsU&t=2s)
+"""
+
+[main_why_teams_love]
+other = 'Why Teams <i class="fa fa-heart"></i> Helm'
+
+[feature_01_title]
+other = "Manage Complexity"
+
+[feature_01_desc]
+other = "Charts describe even the most complex apps, provide repeatable application installation, and serve as a single point of authority."
+
+[feature_02_title]
+other = "Easy Updates"
+
+[feature_02_desc]
+other = "Take the pain out of updates with in-place upgrades and custom hooks."
+
+[feature_03_title]
+other = "Simple Sharing"
+
+[feature_03_desc]
+other = "Charts are easy to version, share, and host on public or private servers."
+
+[feature_04_title]
+other = "Rollbacks"
+
+[feature_04_desc]
+other = "Use `helm rollback` to roll back to an older version of a release with ease."
+
+## homepage get helm
+[get_helm_title]
+other = "Get Helm!"
+
+[get_helm_desc]
+other = "Install Helm with a package manager, or [download a binary](https://github.com/helm/helm/releases/latest)."
+
+[get_helm_next]
+other = "Once installed, unpack the helm binary and add it to your PATH and you are good to go! Check the [docs](./docs/intro/install/) for further [installation](./docs/intro/install/) and [usage instructions](./docs/intro/using_helm/)."
+
+[get_charts_title]
+other = "Get Charts"
+
+[get_charts_desc]
+other = "Visit the [Helm Hub](https://hub.helm.sh) to explore charts from numerous public Helm repositories."
+
+[get_upgrade_title]
+other = "[Upgrading from v2 to v3?](https://helm.sh/docs/topics/v2_v3_migration/)"
+
+[get_upgrade_desc]
+other = "[Read the migration doc for more details.](https://helm.sh/docs/topics/v2_v3_migration/)"
+
+## homepage community
+[community_title]
+other = "Join the Community"
+
+[community_desc]
+other = "More information about the Helm project, and how to contribute."
+
+[community_events_title]
+other = "Events"
+
+[community_upcoming_events_title]
+other = "Upcoming Events"
+
+[community_upcoming_events_desc]
+other = "Watch this space..."
+
+[community_past_events_title]
+other = "Upcoming Events"
```

Comment:
Upcoming --> Past

Reply:
This was in the Chinese one as well. Fixed in both.